src/api/participantsRouter.ts

```diff
@@ -204,3 +183,6 @@ participantsRouter.post(
     return res.status(200).json(sharingParticipants);
   }
 );
+
+export const currentParticipantRouter = express.Router();
+participantsRouter.use('/current', enrichCurrentParticipant, currentParticipantRouter);
```

Comment:
This breaks the convention we have everywhere else, where each endpoint declares the full route under that router type, and uses route params for parts of the REST path (e.g. the participant to query).

Is there any reason this couldn't follow the same convention (and be part of the same router), but with a route parameter that will parse as a string (and expect 'current')? It would fit in with everything else much better.

Reply:
Oh I think I misunderstood the conversation before. I thought endpoint like teammember and business contact will have a separate endpoint like `/participants/current/businessContact`. I think what you said make sense, which will make all current endpoint works without adding more endpoints. I will make a change to that